### PR TITLE
zpool_facts - pythonification

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
Minor refactorning.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
plugins/modules/storage/zfs/zpool_facts.py

### CODE REVIEW

**Comment by mator** [comment]
LGTM

File: plugins/modules/storage/zfs/zpool_facts.py [hunk 1]
```diff
@@ -153,7 +146,7 @@ def get_facts(self):
         if self.name:
             cmd.append(self.name)
 
-        (rc, out, err) = self.module.run_command(cmd)
+        rc, out, err = self.module.run_command(cmd, check_rc=True)
 
         if rc == 0:
```
Comment (felixfontein) [inline plugins/modules/storage/zfs/zpool_facts.py:151]: This should be removed, and the folowing lines dedented.

Comment (russoz) [reply plugins/modules/storage/zfs/zpool_facts.py]: Done

**Comment by felixfontein** [comment]
@russoz thanks for improving this!
@mator thanks for reviewing!

**Pull request merged**